### PR TITLE
[keras/datasets] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/datasets/imdb.py [hunk 1]
```diff
@@ -58,17 +58,17 @@ def load_data(
...
       skip_top: skip the top N most frequently occurring words
           (which may not be informative). These words will appear as
-          `oov_char` value in the dataset. Defaults to 0, so no words are
-          skipped.
+          `oov_char` value in the dataset. When 0, no words are
+          skipped. Defaults to `0`.
       maxlen: int or None. Maximum sequence length.
-          Any longer sequence will be truncated. Defaults to None, which
-          means no truncation.
+          Any longer sequence will be truncated. None, means no truncation.
```
Comment (qlzh727) [inline keras/datasets/imdb.py:67]: Please quote the None here.

File: keras/datasets/reuters.py [hunk 2]
```diff
@@ -65,20 +65,20 @@ def load_data(
...
       skip_top: skip the top N most frequently occurring words
           (which may not be informative). These words will appear as
-          `oov_char` value in the dataset. Defaults to 0, so no words are
-          skipped.
+          `oov_char` value in the dataset. 0 means no words are
+          skipped. Defaults to 0
       maxlen: int or None. Maximum sequence length.
-          Any longer sequence will be truncated. Defaults to None, which
-          means no truncation.
+          Any longer sequence will be truncated. None means no truncation.
```
Comment (qlzh727) [inline keras/datasets/reuters.py:74]: Same.

File: keras/datasets/reuters.py [hunk 3]
```diff
@@ -65,20 +65,20 @@ def load_data(
           ranked by how often they occur (in the training set) and only
           the `num_words` most frequent words are kept. Any less frequent word
           will appear as `oov_char` value in the sequence data. If None,
-          all words are kept. Defaults to None, so all words are kept.
+          all words are kept. Defaults to `None`.
       skip_top: skip the top N most frequently occurring words
           (which may not be informative). These words will appear as
-          `oov_char` value in the dataset. Defaults to 0, so no words are
-          skipped.
+          `oov_char` value in the dataset. 0 means no words are
```
Comment (qlzh727) [inline keras/datasets/reuters.py:71]: Same for 0.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/datasets/reuters.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/datasets/reuters.py:74]: Fixed

File: keras/datasets/reuters.py (hunk 3 above)
Comment (SamuelMarks) [reply keras/datasets/reuters.py:71]: Fixed

File: keras/datasets/imdb.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/datasets/imdb.py:67]: Fixed

**Comment by gbaned** [comment]
Hi @SamuelMarks Can you please resolve conflicts? Thank you!

**Comment by SamuelMarks** [comment]
@gbaned Fixed

**Review by qlzh727 — APPROVED** [review]

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Please fix the lint error.

**Comment by SamuelMarks** [comment]
Fixed

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**